content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -193,6 +193,84 @@ it does the following when a Pod is created:
 1. If the spec of the incoming Pod doesn't already contain any `imagePullSecrets`, then the
    admission controller adds `imagePullSecrets`, copying them from the `ServiceAccount`.
 
+### Legacy ServiceAccount token cleaner
+
+{{< feature-state for_k8s_version="v1.29" state="beta" >}}
+
+Starting from Kubernetes version 1.28, the feature `LegacyServiceAccountTokenTracking`
+is enabled by default, which creates a ConfigMap that records the configmap
+creation date as the tracking date. With the feature enabled, each use of the
+legacy secret token updates the secret label `kubernetes.io/legacy-token-last-used`,
+with the latest usage date as the value.
+
+Starting from Kubernetes version 1.29, the legacy ServiceAccount token cleaner
+is enabled by default. It runs as part of `kube-controller-manager`
+and acts asynchronously. Every 24 hours, the cleaner checks if the current time
+has been a `speficied time` since the ConfigMap tracking date. If true, it loops
+the secret list and evaluates the secrets that have the type `kubernetes.io/service-account-token`.
+If the secret satisfies all the following conditions:
+- is auto-generated (secret bi-directionally referenced by the ServiceAccount)
+- is not mounted by pods
+- is not used in a `specified time` since the last-used/secret-creation date
+
+the cleaner marks the secret as invalid by adding a label `kubernetes.io/legacy-token-invalid-since`,
+with the current date as value. If the invalidated token is not used in a
+`specified time`, it will be cleaned by the cleaner.
+
+{{< note >}}
+All the `specificied time` above is default to be one year. Users can configure
+this value through `--legacy-service-account-token-clean-up-period` flag.
+{{< /note >}}
+
+If users use the invalidated auto-generated legacy token, the legacy token
+validator will
+- add an audit annotation for the key-value pair
+`authentication.k8s.io/legacy-token-invalidated`:`secretName/Namepace`,
+- increment the `invalid_legacy_auto_token_uses_total` metric count,
+- update the secret label `kubernetes.io/legacy-token-last-used` with the new date,
+- return an error indicating that the legacy token has been invalidated.
+
+If received this validation error, users can update the secret to remove the
```

Comment:
Please be careful when spelling "secret" or "Secret".
The former is a generic word and the second is treated as an object of a resource type.

Reply:
Revised.